assets/scss/libs/iconfont/_icon-font.scss

```diff
@@ -60,163 +60,167 @@
     content: "\E009";
   }
 
-  .icon-doppler-logo:before {
+  .icon-doppler-logo-iso:before {

```

Comment:
You are replacing `.icon-doppler-logo` with `.icon-doppler-logo-iso` is it correct? Maybe I'm missing something...